src/sentry/conf/server.py

```diff
@@ -2364,6 +2364,7 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 SENTRY_LLM_OPTIONS: dict[str, str] = {
     "url": os.getenv("LLM_URL", ""),
     "model": os.getenv("LLM_MODEL", ""),
```

Comment:
wondering if we should set the default here, since `SENTRY_AI_SUGGESTED_FIX_MODEL` sets the default